### PR TITLE
RbOnClose claim delay

### PR DESCRIPTION
**Pull request by g4s8**
#616 - set `until` claim field to 15min for close claims from `RbOnClose` reaction.

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#650](https://github.com/zerocracy/farm/issues/650) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#650](https://github.com/zerocracy/farm/issues/650) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/570818761). The budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4). Please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket. If you decide that this PR should not be accepted ever, also inform the architect.

**Comment by yegor256** [comment]
@g4s8 but just setting `until` won't help. The claim will still stay in the list and in 15 minutes will remove the job from the WBS, even though the ticket has been reopened after closing.

**Comment by g4s8** [comment]
@yegor256 sorry, forget to add check to script, please see new commit

File: src/main/resources/com/zerocracy/stk/pm/in/orders/close_job.groovy [hunk 1]
```diff
@@ -16,19 +16,30 @@
...
+
 import java.security.SecureRandom
 
 def exec(Project project, XML xml) {
   new Assume(project, xml).notPmo()
   new Assume(project, xml).type('Close job')
   ClaimIn claim = new ClaimIn(xml)
   String job = claim.param('job')
+  Farm farm = binding.variables.farm
+  Github github = new ExtGithub(farm).value()
```
Comment (yegor256) [inline src/main/resources/com/zerocracy/stk/pm/in/orders/close_job.groovy]: @g4s8 not all jobs are in GitHub. We will have JIRA/Trello/etc jobs in the future. You should check whether the job name starts wih `gh:`

**Comment by yegor256** [comment]
@g4s8 see above

**Comment by g4s8** [comment]
@yegor256 fixed, please look again

**Comment by 0crat** [comment]
@carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda) this job was assigned to you 5 days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by carlosmiranda** [comment]
@yegor256 ping

**Comment by yegor256** [comment]
@rultor merge

**Comment by 0crat** [comment]
Order was finished, quality was "acceptable": +15 points just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)

**Comment by 0crat** [comment]
The job [#650](https://github.com/zerocracy/farm/issues/650) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 points just awarded to @yegor256[/z](https://www.0crat.com/u/yegor256)

**Pull request merged**